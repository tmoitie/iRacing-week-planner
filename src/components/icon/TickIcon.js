import React from 'react';
import styles from '../../styles/main.module.scss';

export default function TickIcon() {
  return <span data-testid="TickIcon" className={`${styles.glyphicon} ${styles['glyphicon-ok']}`} />;
}
