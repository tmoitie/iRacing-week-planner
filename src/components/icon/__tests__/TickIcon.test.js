import React from 'react';
import { render } from '@testing-library/react';
import { describe, test } from '@jest/globals';
import TickIcon from '../TickIcon';

describe('components/icon/TickIcon', () => {
  test('renders correctly', () => {
    const { container } = render(<TickIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
