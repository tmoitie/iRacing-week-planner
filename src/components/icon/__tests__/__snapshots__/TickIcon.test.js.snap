// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/icon/TickIcon renders correctly 1`] = `
<span
  class="glyphicon glyphicon-ok"
  data-testid="TickIcon"
/>
`;
